colin/core/constant.py

```diff
@@ -17,6 +17,7 @@
 RULESET_DIRECTORY_NAME = "rulesets"
 RULESET_DIRECTORY = "share/colin/" + RULESET_DIRECTORY_NAME
 JSON = ".json"
+YAML = [".yaml", '.yml']
```

Comment:
can we instead do here one constant which would indicate acceptable extensions so there is no need of `[JSON] + YAML` in the code below?

Reply:
We can do that but the problem is, I looked around the internet and it seems both extensions are widely used. Which extension should we go with then?

The `[JSON] + YAML` seemed dirty to me too. Another approach to solve it — We can turn JSON into a list too, that way the code would be cleaner, and consistency would be maintained. And we won't have to carry the burden of selecting an extension for `YAML` too that way.

Reply:
@lachmanfrantisek What do you think about this?

Reply:
For me, one constant (=list of strings) with all possible extensions would be better.

Reply:
Yeah that sounds best! I think I'll go with that.

Reply:
I would do `["yml", "yaml", "json"]`